TODO.md

```diff
@@ -0,0 +1,69 @@
+# TODO
+remaining tasks for the `0.2` release, in order of priority
+
+## major
+* [ ] move behaviours (getting, updating, etc.) to library methods
+* [ ] validation (schema, column, relationship)
+* [ ] declarative entity definitions
+* [ ] schema validation
+* [ ] documentation
+* [ ] callbacks
+* [ ] pagination
+* [ ] hateoas-style support
+
+### move behaviours
+at the moment, when you generate your code, a lot of functionality
+gets put in the restplus endpoint. this results in huge diffs between
+versions and is tiresome to test.
+
+it is proposed that this code moves into a library function which then
+gets imported by the endpoint. this will make the code easier to test,
+and will result in a smaller generated codebase.
```

Comment:
Love it. It would allow people to be much more granular about which bits of the Genyrator they use.